sdk/formrecognizer/Azure.AI.FormRecognizer/src/FormTrainingClient.cs

```diff
@@ -216,9 +300,9 @@ public virtual CreateComposedModelOperation StartCreateComposedModel(IEnumerable
                 }
 
                 var composeRequest = new ComposeRequest(modelIdsGuid);
-                if (createComposedModelOptions.ModelName.Length > 0)
+                if (!string.IsNullOrEmpty(modelName))
```

Comment:
just curious, why do we do client validation on the modelName here, but not in StartTraining?

Reply:
Hmm great point. I was using previous logic which made more sense but now it is extra. I will remove

Reply:
Thank you!
Updated